### ISSUE TITLE
Change path for Ubuntu docker GPG key

### ISSUE THREAD

**Issue by dogsbody** [docker/docs#17471]
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

Information is incorrect

### Description

I suggest changing instances of `/etc/apt/keyrings/docker.gpg` to `/usr/share/keyrings/docker.gpg`

The reasons:

1. The instructions were asking us to create `/etc/apt/keyrings` while `/usr/share/keyrings` already exists for exactly this use.
2. The old path is very close to `/etc/apt/trusted.gpg.d/` which should **never** be used.  Any gpg keys placed in `/etc/apt/trusted.gpg.d` will be trusted blindly by all repositories on the system.

In short, in the last few years, `/usr/share/keyrings` has now been defined as the place to put these new repository specific gpg keys :+1:

### Location

https://docs.docker.com/engine/install/ubuntu/

**Comment by dvdksn** [comment]
The keyrings directory won't be changed, details here: https://github.com/docker/docs/pull/17686#issuecomment-1622150623
